Description: Try to reduce random test failures

Author: Rebecca N. Palmer <rebecca_palmer@zoho.com>
Bug-Debian: https://bugs.debian.org/1030096
Forwarded: no

--- dask.distributed.orig/distributed/tests/test_nanny.py
+++ dask.distributed/distributed/tests/test_nanny.py
@@ -218,7 +218,7 @@
     with captured_logger(
         logging.getLogger("distributed.nanny"), level=logging.ERROR
     ) as logger:
-        await a.restart(timeout=0.1)
+        await a.restart(timeout=0.001)
 
     out = logger.getvalue()
     assert "timed out" in out.lower()
--- dask.distributed.orig/distributed/tests/test_utils_test.py
+++ dask.distributed/distributed/tests/test_utils_test.py
@@ -959,7 +959,7 @@
                 ),
             ],
             capture_output=True,
-            terminate_timeout=1,
+            terminate_timeout=10,
         ) as proc:
             assert proc.stdout
             assert proc.stdout.readline().strip() == b"ready"
